src/test/java/com/glovoapp/planningpoker/e2e/voting/Median2UsersTest.java

```diff
@@ -0,0 +1,60 @@
+package com.glovoapp.planningpoker.e2e.voting;
+
+import com.glovoapp.planningpoker.e2e.WebDriverFactory;
+import com.glovoapp.planningpoker.e2e.pages.MainPage;
+import com.glovoapp.planningpoker.e2e.pages.SessionPage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.Optional;
+
+import static com.glovoapp.planningpoker.e2e.WebDriverFactory.getDriver;
+
+public class Median2UsersTest {
+    private final WebDriver driver = getDriver();
+    private final WebDriver driver2 = getDriver();
+
+    MainPage firstUserMainPage = new MainPage(driver);
+    SessionPage firstUserSessionPage = new SessionPage(driver);
+    MainPage secondUserMainPage = new MainPage(driver2);
```

Comment:
Field should be `private final`.

```suggestion
    private final MainPage secondUserMainPage = new MainPage(driver2);
```